tests/objcthemis/objthemis/SecureMessageTests.m

```diff
@@ -96,16 +96,27 @@ - (void)wrapUnwrapWithPrivateKey:(NSData *)privateKeyData publicKey:(NSData *)pu
     NSString *resultString = [[NSString alloc] initWithData:decryptedMessage encoding:NSUTF8StringEncoding];
     XCTAssertTrue([message isEqualToString:resultString], @"initial string and decrypted string should be the same");
   
+    // NULL message
+    NSError *nullErrorEn = nil;
+    NSData *nullMessageEn = [encrypter wrapData:NULL error:&nullErrorEn];
+    XCTAssertNotNil(nullErrorEn, @"encrypting data with NULL message is error");
+    XCTAssertNil(nullMessageEn, @"encrypting data with NULL message should return nil message");
+    
+    NSError *nullErrorDe = nil;
+    NSData *nullMessageDe = [encrypter unwrapData:NULL error:&nullErrorDe];
+    XCTAssertNotNil(nullErrorDe, @"decrypting data with NULL message is error");
+    XCTAssertNil(nullMessageDe, @"decrypting data with NULL message should return nil message");
+    
     // empty message
-    NSError *themisEmptyErrorEn = nil;
-    NSData *emEmptyMessageEn = [encrypter wrapData:NULL error:&themisEmptyErrorEn];
-    XCTAssertNotNil(themisEmptyErrorEn, @"encrypting data with empty message is error");
-    XCTAssertNil(emEmptyMessageEn, @"encrypting data with empty message should return nil message");
-  
-    NSError *themisEmptyErrorDe = nil;
-    NSData *emEmptyMessageDe = [encrypter unwrapData:NULL error:&themisEmptyErrorDe];
-    XCTAssertNotNil(themisEmptyErrorDe, @"decrypting data with empty message is error");
-    XCTAssertNil(emEmptyMessageDe, @"decrypting data with empty message should return nil message");
+    NSError *emptyErrorEn = nil;
+    NSData *emptyMessageEn = [encrypter wrapData:[@"" dataUsingEncoding:NSUTF8StringEncoding] error:&emptyErrorEn];
+    XCTAssertNotNil(emptyErrorEn, @"encrypting data with empty message is error");
+    XCTAssertNil(emptyMessageEn, @"encrypting data with empty message should return nil message");
+    
+    NSError *emptyErrorDe = nil;
+    NSData *emptyMessageDe = [encrypter unwrapData:NULL error:&emptyErrorDe];
```

Comment:
Well, this isn't an empty message ;)

I guess it's better to have it like this

```obj-c
NSData *emptyData = [NSData data];
```

and then use it everywhere we need an empty message or a key.